site/content/news/performance-in-3.2.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Performance improvements in RDF4J 3.2"
+authors:
+  - havard
+  - jeen
+# set correct date when finalized
+date: 2020-06-02 
+layout: "single"
+categories: ["news"]
+---
+<div class="big-emoji">&#x1F680;</div>
+
+The release of [RDF4J 3.2.0](/release-notes/3.2.0/) introduced a large number
+of performance improvements to the framework. 
+
+One major change was the introduction of a new `Model` implementation, the
+`DynamicModel`, and switching to this new model implementation in the
+transaction isolation handling modules, in particular for the native and memory
+stores.
+
+The DynamicModel has a large effect on the transaction isolation overhead in
+the MemoryStore. Typical transaction isolation added roughly 100% overhead when
+adding data to the store, with the introduction of the DynamicModel this has
+been reduced to 25% as long, as there are no queries that cause the DynamicModel
+to upgrade to a full LinkedHashModel.
+
+# TODO explain what kind of queries would cause the upgrade.
```

Comment:
There are only two kinds of queries that will not force the DynamicModel to upgrade. Those are:

 - all statements from all contexts
 - single statement (fixed subject, predicate, object, context)

Reply:
minor typo on line 24 / comma in the wrong place: "25% as long, as"  should probably read "25%, as long as"